source/anonymize/index.rst

```diff
@@ -0,0 +1,1309 @@
+Anonymize
+=========
+
+Requirements (on-premise)
+-------------------------
+
+============ =========== ===========
+GLPI Version Minimum PHP Recommanded
+============ =========== ===========
+10.0.x       8.1         8.2
+============ =========== ===========
+
+Install the plugin
+------------------
+
+.. Note::
+   A `basic licence <https://services.glpi-network.com/#offers>`__ (or higher) is required. This plugin is also available from the `Cloud <https://glpi-network.cloud/fr/>`__.
+
+-  Go to the marketplace. Download and install the “**Anonymize**”   plugin.
+
+.. figure:: images/Anonymize-1.png
+   :alt: Install the plugin
+
+Choice of profile
+-----------------
+
+Anonymisation works by profile. There are 4 profiles already created.
+When you select a profile, a certain amount of data is already set. They can, of course, be modified.
+
+-  Go to **administration** > **anonymization profile** and select the profile best suited to your needs:
+
+The 4 profiles you will find are :
+
+-  Email,
+-  Entity,
+-  Computer
+-  User profile
+
+.. figure:: images/Anonymize-2.png
+   :alt: Profiles
+
+Anonymisable data per profile (predefined)
+------------------------------------------
+
+Each profile has an element assigned to it. You will need to define the value of each attribute that it will be able to anonymise (or not)
+
+.. figure:: images/Anonymize-3.png
+   :alt:
+
+Email
+~~~~~
+
+- **users_id**
```

Comment:
All lists entries in bold looks like strange
```suggestion
- users_id
```

Or maybe:
```suggestion
- `users_id`
```

Reply:
I agree, i can change modify by `users_id`